src/main/c/ssl.c

```diff
@@ -2036,6 +2036,17 @@ TCN_IMPLEMENT_CALL(void, SSL, setState)(TCN_STDARGS,
     SSL_set_state(ssl_, state);
 }
 
+TCN_IMPLEMENT_CALL(jint, SSL, release0)(TCN_STDARGS)
```

Comment:
Will something like the following be sufficient? With the current system we may be masking useful information related to the specific version within each implementation of the OpenSSL interface.

```c
TCN_IMPLEMENT_CALL(jint, SSL, versionAsCompiled)(TCN_STDARGS) {
  return SSLEAY_VERSION;
}
```